README.md

```diff
@@ -170,9 +170,9 @@ The `-r` option can be used to specify a text file listing packages to be instal
 
 List or change working virtual environments.
 
-`usage: pew workon [environment_name] [--here]`
+`usage: pew workon [-h] [-n] [envname]`
```

Comment:
As pfmoore mentioned, can you please use the long form of the option?

Otherwise, it looks good